scripts/pylib/twister/twisterlib/reports.py

```diff
@@ -36,7 +36,7 @@ class Reporting:
             'deny_suite': ['footprint']
         },
         'footprint.json': {
-            'deny_status': ['FILTER'],
+            'deny_status': ['filtered'],
```

Comment:
this change leads to `KeyError: 'filtered'` without the appropriate changes here

https://github.com/zephyrproject-rtos/zephyr/blob/52540d8412630edb2c64f4fcca1c1165bcce1ac3/scripts/pylib/twister/twisterlib/reports.py#L289

https://github.com/zephyrproject-rtos/zephyr/blob/52540d8412630edb2c64f4fcca1c1165bcce1ac3/scripts/pylib/twister/twisterlib/reports.py#L294